Gruntfile.js

```diff
@@ -239,7 +240,7 @@ module.exports = function(grunt) {
 
     githooks: {
       all: {
-        'pre-commit': 'jshint cover'
+        'pre-commit': 'jshint'
```

Comment:
I think it was meant to, but shouldn't go in. I misunderstood @csbailey5t. The githook/test process needs some love, but in the meantime, if you need to commit something with failing test or fallen coverage to compare against travis, just `rm .git/hooks/pre-commit`. It will be re-added automatically on the next `npm install`.